Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3.8
+LABEL maintainer "Brian Walsh <walsbr@ohsu.edu>"
+# ensure OpenSSL installed
+RUN    apt-get update \
+    && apt-get install openssl \
+    && apt-get install ca-certificates
+
+# this app has an example backend that calls ensemble
+#
+# snafu!: ensemble's certificate has a problem
+# see https://github.com/Ensembl/ensembl-rest/issues/427
+#
+# fix it by downgrading SSL default
+RUN sed -i 's/DEFAULT@SECLEVEL=2/DEFAULT@SECLEVEL=1/' /usr/lib/ssl/openssl.cnf
```

Comment:
remove this since we are not using ensemble REST

Reply:
Done